apps/desktop-wallet/src/pages/NewWallet/CreateWalletPage.tsx

```diff
@@ -77,10 +77,8 @@ const CreateWalletPage = ({ isRestoring = false }: { isRestoring?: boolean }) =>
     sendAnalytics({ event: 'Creating wallet: Creating password: Clicked next' })
 
     try {
-      const newWalletId = saveNewWallet({ walletName, encrypted: await encryptMnemonic(mnemonic, password) })
+      saveNewWallet({ walletName, encrypted: await encryptMnemonic(mnemonic, password) })
       resetCachedMnemonic()
-      clearQueryCache()
-      await restoreQueryCache(newWalletId)
```

Comment:
There's no point in restoring a query cache when we create a new wallet which by definition is new and has nothing to restore.